social-network/src/CreativityPage/PostDate.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class PostDate extends Component {
```

Comment:
Great job using modular components! Really important, especially with this `PostDate` component that gets a little tricky when it comes to rendering the date.